Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -553,20 +554,21 @@ void MatchTOF::propagateTPCTracks(int sec)
     }
 
     if (trc.getX() < o2::constants::geom::XTPCOuterRef - 1.) {
-      if (!propagateToRefX(trc, o2::constants::geom::XTPCOuterRef, 10, intLT0) || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagat>
+      if (!propagateToRefX(trc, o2::constants::geom::XTPCOuterRef, 10, intLT0dummy) || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagat>
         mNotPropagatedToTOF[trkType::UNCONS]++;
         continue;
       }
     }
 
-    o2::base::Propagator::Instance()->estimateLTFast(intLT0, trc);
-
     // the "rough" propagation worked; now we can propagate considering also the cov matrix
-    if (!propagateToRefX(trc, mXRef, 2, intLT0)) { // || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagation with the cov matrix w>
+    if (!propagateToRefX(trc, mXRef, 2, intLT0dummy)) { // || TMath::Abs(trc.getZ()) > Geo::MAXHZTOF) { // we check that the propagation with the cov matrix w>
```

Comment:
leave the original estimateLTFast (till the starting point) and propagate with `intLT0`: the latter is more precise when propagation in the material is involved.

Reply:
done but now using propagateToRefXWithoutCov mehod

Reply:
I'll resolve the comment, once you have read it

Reply:
sure, done